frontend/public/locales/en/translation.json

```diff
@@ -1103,12 +1105,12 @@
     "empty-state.node.title": "No nodes found",
     "empty-state.submariner.title": "No submariner add-ons found",
     "emptystate.addCredential": "Add credential",
-    "emptystate.credentials.msg": "You don't have any discovered clusters. You have {{discoveryConfigTotal}} credentials. Click the <bold>Configure Discovery</bold> button to set up a filter for discovered clusters from your connections.",
-    "emptystate.credentials.title": "No discovered clusters found",
-    "emptystate.defaultState.msg": "You don't have any discovered clusters. Credentials of <bold>Red Hat OpenShift Cluster Manager</bold> type must be added to configure Discovery.",
-    "emptystate.defaultState.title": "No discovered clusters found",
-    "emptystate.discoveryEnabled.msg": "You don't have any discovered clusters. Cluster discovery was configured. Return to this page later, configure Discovery again, or <a>view documentation <icon /></a>",
-    "emptystate.discoveryEnabled.title": "No discovered clusters found",
+    "emptystate.credentials.msg": "No discovered clusters found. You have {{discoveryConfigTotal}} credentials. Click the <bold>Configure Discovery</bold> button to set up a filter for discovered clusters from your connections.",
+    "emptystate.credentials.title": "You don't have any discovered clusters",
+    "emptystate.defaultState.msg": "Credentials of <italic>Red Hat OpenShift Cluster Manager</italic> type must be added to configure Discovery.  Click the <bold>Add credential</bold> button to add credential.",
```

Comment:
"Click Add credential to add credential" seems a bit redundant

Reply:
@KevinFCormier - I completely agree - it felt strange typing it out.  Will definitely heed the advice from @swopebe and doc team to update these.